python/semantic_kernel/agents/open_ai/responses_agent_thread_actions.py

```diff
@@ -705,22 +735,16 @@ def _prepare_chat_history_for_request(
                             )
 
                         contents.append({"type": "input_image", "image_url": image_url})
-                    case FunctionCallContent():
-                        fc_dict = {
-                            "type": "function_call",
-                            "id": content.id,
-                            "call_id": content.call_id,
-                            "name": content.name,
-                            "arguments": content.arguments,
-                        }
-                        response_inputs.append(fc_dict)
                     case FunctionResultContent():
                         rfrc_dict = {
                             "type": "function_call_output",
                             "output": str(content.result),
                             "call_id": content.call_id,
                         }
                         response_inputs.append(rfrc_dict)
+                    case FunctionCallContent():
```

Comment:
This isn't the case when `store_enabled` is False. We need to build up the FCC based on the chat history. Please revert this change, I'm handling it in a different PR.